Gopkg.lock

```diff
@@ -419,6 +418,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "126ca98597875822b477375d68ee4318150c6ebd9faf1dd5ab368fbe1fe99e9d"
+  inputs-digest = "f41ce71fcc26465ebf6a3b4171311624087e86431cd519fd26c839c4434ab180"
```

Comment:
@anjannath Not sure why you need to change the digest.

Reply:
this happened because of the `registry` package. the function `registry.OpenKey()` is undefined in non windows OS, so i did a `dep ensure` to update the dependency thinking it was version issue. 
I can revert the changes in the file if needed.

Reply:
We need to run platform specific tests? If so, define these as a new function which can be overriden by a platform specific source file.

Reply:
I did that, at least i think i did that, waiting for your review.

Reply:
-1, this change is a change in a generated file. You cannot edit the lock file. You need to modify Gopkg.toml and make sure dependency resolution will work.